subscriber/src/main.rs

```diff
@@ -0,0 +1,66 @@
+
+// Copyright 2018 Bitwise IO
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#[macro_use]
+extern crate clap;
+#[macro_use]
+extern crate log;
+extern crate simple_logger;
+extern crate postgres;
+use log::LogLevel;
+
+use postgres::{Connection, TlsMode};
+
+mod database;
+use database::Database;
+
+fn main() {
+    let matches = clap_app!(sawtooth_bond_subscriber =>
+        (version: crate_version!())
+        (about: "Sawtooth Bond Subscriber")
+        (@arg connect: -C --connect +takes_value
+           "connection endpoint for validator")
+        (@arg verbose: -v --verbose +multiple
+           "increase output verbosity")
+        (@arg dbname: default_value("sawtooth-bond") --db_name +takes_value
+           "the name of the database")
+        (@arg dbhost: default_value("localhost") --db_host +takes_value
+            "the host of the database")
+        (@arg dbport: default_value("5432") --db_port +takes_value
+            "the host of the database")
```

Comment:
Small typo, should be "the port of the database"

Reply:
Fixed :)